openshift/e2e-tests-openshift.sh

```diff
@@ -25,6 +25,44 @@ SERVICE_ACCOUNT=builder
 # Install CI
 [[ -z ${LOCAL_CI_RUN} ]] && install_pipeline_crd
 
+# Pipelines Catalog Repository
+PIPELINES_CATALOG_URL=https://github.com/openshift/pipelines-catalog/
+PIPELINES_CATALOG_REF=origin/master
+PIPELINES_CATALOG_DIRECTORY=./openshift/pipelines-catalog
+PIPELINES_CATALOG_IGNORE="s2i-dotnet-1"
+PIPELINES_CATALOG_PRIVILIGED_TASKS="s2i-*"
+
+# Add PIPELINES_CATALOG in here so we can do the CI all together.
+# We checkout the repo in ${PIPELINES_CATALOG_DIRECTORY}, merge them in the main
+# repos and launch the tests.
+function pipelines_catalog() {
+    set -x
+    local oldRepoRootDir
+    local ptest parent
+
+    [[ -d ${PIPELINES_CATALOG_DIRECTORY} ]] || \
+        git clone --depth=1 ${PIPELINES_CATALOG_URL} ${PIPELINES_CATALOG_DIRECTORY}
+
+    pushd ${PIPELINES_CATALOG_DIRECTORY} >/dev/null && \
+        git reset --hard ${PIPELINES_CATALOG_REF} &&
+        popd >/dev/null
+
+    # NOTE(chmouel): The functions doesnt support argument so we can't just leave the test in
+    # ${PIPELINES_CATALOG_DIRECTORY} we need to have it in the top dir, TODO: fix the functions
+    for ptest in ${PIPELINES_CATALOG_DIRECTORY}/*/tests;do
+        parent=$(dirname ${ptest})
+        base=$(basename ${parent})
+        in_array ${base} ${PIPELINES_CATALOG_IGNORE} && { echo "Skipping: ${btest}"; continue ;}
```

Comment:
shouldn't it be `${ptest}` instead of `${btest}`

Reply:
ah yes it's `base`. that should have been there,

thanks,